### PR TITLE
README.md with build process for linux/osx, go.mod

### PR DESCRIPTION
**Pull request by ofen**

### CODE REVIEW

**Comment by ofen** [comment]
@cryptonote-social hi! Could you please take a look?

**Comment by cryptonote-social** [comment]
thanks for your contribution, I will get to this shortly!

File: miner.go [hunk 1]
```diff
@@ -57,6 +57,9 @@ func Mine(c *MinerConfig) error {
 		ExcludeHourStart: c.ExcludeHrStart,
 		ExcludeHourEnd:   c.ExcludeHrEnd,
 	})
+	if imResp.Code < 0 {
```
Comment (cryptonote-social) [inline miner.go:60]: nice catch! would you also add a crylog.Error() here before returning to be consistent with the other error handling in this method?

**Review by cryptonote-social — CHANGES_REQUESTED** [review]

**Comment by ofen** [comment]
Branches synced

File: miner.go (hunk 1 above)
Comment (ofen) [reply miner.go:61]: crylog added

**Pull request merged**